pom.xml

```diff
@@ -50,11 +57,6 @@
                 <artifactId>guava</artifactId>
                 <version>${version.com.google.guava.guava}</version>
             </dependency>
-            <dependency>
```

Comment:
@joaofreitas-qubit why are we removing the dependency itself and not just the version?